examples/python/zarrv2-raw-filesystem.py

```diff
@@ -0,0 +1,55 @@
+# Basic Zarr V2 to filesystem
+import numpy as np
+from acquire_zarr import (
+    StreamSettings, ZarrStream, Dimension, DimensionType, ZarrVersion, DataType
+)
+
+
+def main():
+    # Configure stream settings
+    settings = StreamSettings()
+
+    # Configure dimensions (t, y, x)
+    settings.dimensions.extend([
+        Dimension(
+            name="t",
+            kind=DimensionType.TIME,
+            array_size_px=0,  # Unlimited
+            chunk_size_px=32,
+            shard_size_chunks=1,
+        ),
+        Dimension(
+            name="y",
+            kind=DimensionType.SPACE,
+            array_size_px=48,
+            chunk_size_px=16,
+            shard_size_chunks=1,
+        ),
+        Dimension(
+            name="x",
+            kind=DimensionType.SPACE,
+            array_size_px=64,
+            chunk_size_px=32,
+            shard_size_chunks=1,
+        ),
+    ])
+
+    settings.store_path = "output_v2.zarr"
+    settings.version = ZarrVersion.V2
+    settings.data_type = DataType.INT32
+
+    # Create stream
+    stream = ZarrStream(settings)
+
+    # Create and write sample data
+    for i in range(10):
```

Comment:
This is just a (nit-picky) suggestion, that we should discuss:
For examples, it might make it more intuitive to show the calls closer to how they actually would be used.  Here, you generate the data in the same loop as you write it.  It might it more intuitive to generate all the data up front and then have only the append (which indexes into an array) inside the loop.   Another possibility would be to put the image generation call inside a get_image_data function call (which gets called within the loop).

Reply:
This comment applies to all the python examples (won't repeat for each)